public/index.html

```diff
@@ -56,11 +39,21 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Eden - Learning tailored for your child's needs</title>
-  </head>
-  <body>
-    <div id="root"></div>
-    <!--
+  <title>Eden - Learning tailored for your child's needs</title>
+</head>
+
+<body>
+  <!-- The core Firebase JS SDK is always required and must be listed first -->
+  <script src="/__/firebase/7.14.2/firebase-app.js"></script>
```

Comment:
Can we get it off of npm? and remove these script tags?
In the index.js file import the following:
```
 import * as firebase from "firebase/app"; 
 import "firebase/analytics";
```